ts/models/conversation.ts

```diff
@@ -144,6 +145,7 @@ export interface ConversationAttributesOptionals {
   triggerNotificationsFor?: ConversationNotificationSettingType;
   isTrustedForAttachmentDownload?: boolean;
   isPinned: boolean;
+  isApproved: boolean;
```

Comment:
make this thing optional on this one otherwise we have to set it everywhere we create a conversation